apps/web/utils/indexeddb/index.ts

```diff
@@ -0,0 +1,31 @@
+"use client";
+
+import { openDB, IDBPDatabase } from "idb";
```

Comment:
is this still in use?

Reply:
No @elie222 , these are not in use. I think these came to this PR when i created a new branch from idb-playground brach.